src/pages/_app.tsx

```diff
@@ -1,21 +1,31 @@
 import { AppProps } from 'next/app';
+import DefualtLayout from '../layout/DefaultLayout';
```

Comment:
`DefaultLayout` is misspelled here (but then correctly spelled in the file name from which it's imported).